libvips/iofuncs/stream.c

```diff
@@ -382,13 +385,6 @@ vips_stream_input_class_init( VipsStreamInputClass *class )
 		G_STRUCT_OFFSET( VipsStreamInput, blob ),
 		VIPS_TYPE_BLOB );
 
-	VIPS_ARG_BOOL( class, "seekable", 4, 
```

Comment:
Was this intentionally removed? It will break:
```c
VipsStreamInput *input = vips_stream_input_new_from_options( "seekable=1" );
```
```
unable to make VipsStreamInput
stream_input: no property named `seekable'
```

Reply:
Hiya, yes, it tests descriptors for seekable automatically now, so you don't need to tell it.

Reply:
I see. I'll try the subclass approach in C#.

Do you know how I can do that easily? For e.g. I don't have access to macros, so `G_DEFINE_TYPE` will not work. I guess pyvips has the same problem, a `vips_stream_build` function would be useful to have for language bindings.

Reply:
Oh huh you're right, you really need C/C++ to subclass. You can sort-of do it in Python, but it's a pretty awful hack.

Let's add a set of callbacks as per your patch ^^ up there.

Reply:
Of course we have to focus on the other things first, this is a problem for later.

Reply:
OK, added another point to the checklist!